packages/application-shell/src/components/authenticated/am-i-logged-in.tsx

```diff
@@ -11,7 +11,7 @@ import AmILoggedInQuery from './authenticated.mc.graphql';
 
 type RenderFnArgs = { isAuthenticated: boolean };
 type Props = {
-  render: (args: RenderFnArgs) => JSX.Element;
+  render: (args: RenderFnArgs) => React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.